src/main/java/edu/cornell/kfs/sys/dataaccess/impl/DynamoDBClient.java

```diff
@@ -0,0 +1,21 @@
+package edu.cornell.kfs.sys.dataaccess.impl;
+
+import com.amazonaws.auth.AWSCredentials;
+import com.amazonaws.auth.BasicAWSCredentials;
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDBClient;
+import edu.cornell.kfs.sys.dataaccess.DocumentstoreClient;
+
+public class DynamoDBClient implements DocumentstoreClient {
+
+    private AmazonDynamoDBClient dynamoDBClient;
+
+    public DynamoDBClient(String host, int port, String dbName, String username, String password) {
+        AWSCredentials credentials = new BasicAWSCredentials(username, password);
+        dynamoDBClient = new AmazonDynamoDBClient(credentials);
+        dynamoDBClient.setEndpoint("http://" + host + ":" + port);
```

Comment:
Are we always expecting to use "http" endpoints here, or do we need to be able to reach "https" endpoints as well?